executor/common_kvm_arm64_syzos.h

```diff
@@ -39,9 +40,16 @@ struct api_call_code {
 	uint32 insns[];
 };
 
+struct api_call_smc {
+	struct api_call_header header;
+	uint32 smc_id;
+	uint64 params[5];
+} __attribute__((packed));
```

Comment:
Why do we need packed? Can we remove it in both places?

Reply:
Not really, I'll remove it.

I am assuming [packed] here doesn't apply recursively to each struct:

```
syzos_api_smc {
        call            const[3, int64]
        size            bytesize[parent, int64]
        arg_id          flags[kvm_smc_id, int32]
        arg_params      array[int64, 5]
} [packed]
```
?

Reply:
For these things we should match C semantics of corresponding constructs.
This allows to mechanically translate C structs, rather than apply some non-obvious reasoning.